playbooks/operationalizing/monitoring_guide.adoc

```diff
@@ -0,0 +1,99 @@
+---
+---
+= Monitoring OpenShift
+Eric Sauer <esauer@redhat.com>
+v1.0, 2017-08-29
+:scripts_repo: https://github.com/redhat-cop/openshift-toolkit
+:toc: macro
+:toc-title:
+
+OpenShift Monitoring is an ever evolving problem space, with many layers, approaches, and complexities. We attempt to unpack them here.
+
+toc::[]
+
+== Overview
+
+NOTE: Before reading this guide, we recommend first reading link:/playbooks/installation/admin_overview.html[An Overview of OpenShift for System Admins].
+
+The following document intends to provide starting guidance on how to build a monitoring approach for OpenShift. In it, we will present a suggested categorization of _checks_ that would form the basis for items that should usually generate an alert of some kind in a production cluster.
+
+This document will not propose any specific tooling for a monitoring approach, but instead presents the data points that are believed to be important when designing a cluster monitoring approach and provides examples of the logic involved in creating alerts.
+
+NOTE: Discussion of tooling and building a monitoring stack implementation will be discussed in a future guide.
+
+Each section below will present a table of data, giving a description of the data point, and a sample command that might trigger an alert.
+
+== Ensuring a cluster is healthy
+
+=== Docker
+
+Docker is an essential component of an OpenShift environment. The overall health of docker on each master and node instance ensures stability within an OpenShift cluster. The following components are areas that should be monitored on each node.
+
+.Docker checks
+[width="100%",cols="3,6,10",options="header"]
+|=========================================================
+|Check Name |Description |Sample Alerting Logic
+
+|Docker Daemon |Check that docker is running on a system | `systemctl is-active docker`
+
+|Docker Storage|Check that docker's local volume is not full | `echo "$(lvs \| awk '/docker-pool/ {print $5}') > 70" \| bc`
```

Comment:
Although this works, I'd also recommend looking at the output from the `docker info` command - i.e.: as done here for Nagios:
https://github.com/redhat-cop/casl-ansible/blob/master/roles/nagios-target/files/plugins/check_docker_storage